LayoutTests/fast/scrolling/mac/scrollbars/rtl-scrollbar-state.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ MockScrollbarsEnabled=false AsyncOverflowScrollingEnabled=true ] -->
+<html>
+<head>
+    <style>
+        body {
+            height: 1000px;
+        }
+        .scroller {
+            margin: 10px;
+            width: 200px;
+            height: 200px;
+            border: 1px solid black;
+            overflow: auto;
+        }
+        .contents {
+            width: 100%;
+            height: 200%;
+        }
+    </style>
+    <script src="../../../../resources/js-test-pre.js"></script>
```

Comment:
Would be nice to use `js-test.js` in new tests instead of the legacy `js-test-pre.js` / `js-test-post.js`.

Reply:
Fixed.